src/sdkV2.ts

```diff
@@ -21,9 +22,12 @@ export default class CegaEvmSDKV2 {
 
   private _treasuryAddress: EvmAddress;
 
+  private _pythAdapterAddress: EvmAddress;
+
   constructor(
     addressManager: EvmAddress,
     treasuryAddress: EvmAddress,
+    pythAdapterAddress: EvmAddress,
```

Comment:
`pythAdapterAddress` will almost never be used by:
- cega-client
- 3rd party sdk users

To make it simpler for most callers, can you make this optional, and put it after the signer so we don't have to be concerned about not including it when we instantiate the SDK